parametric/apps/nodejs/servicer.js

```diff
@@ -26,11 +26,11 @@ class Servicer {
             parent.origin = request.origin;
         }
 
-        const { http_headers = {} } = request.http_headers || {};
+        const http_headers = request.http_headers.http_headers || [];
         // Node.js HTTP headers are automatically lower-cased, simulate that here.
         const convertedHeaders = {};
-        for (const [key, value] of Object.entries(http_headers)) {
-            convertedHeaders[key.toLowerCase()] = value;
+        for (const header_tuple of http_headers) {
+            convertedHeaders[header_tuple.key.toLowerCase()] = header_tuple.value;
```

Comment:
```suggestion
        for (const { key, value } of http_headers) {
            convertedHeaders[key.toLowerCase()] = value;
```

Reply:
Nice destructuring, thanks for the suggestion! Done in f7f2b19244143ecb77c63ac1fa090bda70492387